Tenduke.Client.WinForms/Tenduke.Client.WinForms.csproj

```diff
@@ -49,8 +49,8 @@
     <None Include="..\10duke-192x192.png" Pack="true" Visible="false" PackagePath="\" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="chromiumembeddedframework.runtime.win-x64" Version="129.0.11" />
-    <PackageReference Include="chromiumembeddedframework.runtime.win-x86" Version="129.0.11" />
+    <PackageReference Include="chromiumembeddedframework.runtime.win-x64" Version="135.0.17" />
+    <PackageReference Include="chromiumembeddedframework.runtime.win-x86" Version="135.0.17" />

```

Comment:
Same comment as for WPF, should ARM be included here?

Reply:
True, I have included for ARM architecture and removed from the sample as this dependency is brought from here.